docs/user/editor/patterns.md

```diff
@@ -37,22 +37,23 @@ Click `Edit schedules` to begin creating or editing trips/frequencies for a trip
 ## Stop sequence
 ### Adding stops
 To begin constructing a pattern, click `+ Add stop`. You can then search for stops to add by name in the dropdown that appears or zoom into the map and add stops by clicking on them and clicking `+ Add stop`.
-<div class="img-center">
-![add stop from dropdown](../../img/pattern-add-stop.png)
+
+<div style="text-align:center">
```

Comment:
Please remove the spaces around `=`. Also, the image tag should be `<img src="path/to/image" />` (not ending with `</img>`).